openshift/clowder/clowd-app.yaml

```diff
@@ -1,82 +1,88 @@
-apiVersion: cloud.redhat.com/v1alpha1
-kind: ClowdApp
+apiVersion: v1
+kind: Template
 metadata:
   name: automation-hub
-  namespace: automation-hub
-spec:
-  # The name of the ClowdEnvironment providing the services
-  envName: automation-hub-env
+objects:
 
-  # The bulk of your App. This is where your running apps will live
-  deployments:
-  - name: galaxy-api
-    podSpec:
-      image: quay.io/cloudservices/automation-hub-galaxy-ng
-      args: ['run', 'api']
-      env:
-        - name: GALAXY_AUTHENTICATION_CLASSES
-          value: '["galaxy_ng.app.auth.auth.RHIdentityAuthentication"]'
-        - name: GALAXY_DEPLOYMENT_MODE
-          value: 'insights'
-        - name: PULP_CONTENT_ORIGIN
-          value: 'localhost'
-        - name: WITH_MIGRATIONS
-          value: '1'
-  - name: pulp-resource-manager
-    podSpec:
-      image: quay.io/cloudservices/automation-hub-galaxy-ng
-      args: ['run', 'resource-manager']
-      env:
-        - name: GALAXY_AUTHENTICATION_CLASSES
-          value: '["galaxy_ng.app.auth.auth.RHIdentityAuthentication"]'
-        - name: GALAXY_DEPLOYMENT_MODE
-          value: 'insights'
-        - name: PULP_CONTENT_ORIGIN
-          value: 'localhost'
-  - name: pulp-content-app
-    podSpec:
-      image: quay.io/cloudservices/automation-hub-galaxy-ng
-      args: ['run', 'content-app']
-      env:
-        - name: GALAXY_AUTHENTICATION_CLASSES
-          value: '["galaxy_ng.app.auth.auth.RHIdentityAuthentication"]'
-        - name: GALAXY_DEPLOYMENT_MODE
-          value: 'insights'
-        - name: PULP_CONTENT_ORIGIN
-          value: 'localhost'
-  - name: pulp-worker
-    podSpec:
-      image: quay.io/cloudservices/automation-hub-galaxy-ng
-      args: ['run', 'worker']
-      env:
-        - name: GALAXY_AUTHENTICATION_CLASSES
-          value: '["galaxy_ng.app.auth.auth.RHIdentityAuthentication"]'
-        - name: GALAXY_DEPLOYMENT_MODE
-          value: 'insights'
-        - name: IMPORTER_MEMORY_REQUEST
-          value: 1Gi
-        - name: IMPORTER_MEMORY_LIMIT
-          value: 2Gi
-        - name: IMPORTER_CPU_REQUEST
-          value: 500m
-        - name: IMPORTER_CPU_LIMIT
-          value: 1000m
-        - name: PULP_CONTENT_ORIGIN
-          value: 'localhost'
+- apiVersion: cloud.redhat.com/v1alpha1
+  kind: ClowdApp
+  metadata:
+    name: automation-hub
+    namespace: automation-hub
+  spec:
+    # The name of the ClowdEnvironment providing the services
+    envName: automation-hub-env
 
-  # Creates a Service on port 8000
-  webServices:
-    public: true
-    private: false
-    metrics: false
+    # The bulk of your App. This is where your running apps will live
+    deployments:
+    - name: galaxy-api
+      podSpec:
+        image: quay.io/cloudservices/automation-hub-galaxy-ng
```

Comment:
You would want to parameterized the image tag e.g. 
`image: quay.io/cloudservices/automation-hub-galaxy-ng:${IMAGE_TAG}`